src/ui/widgets/Shape/shape.tsx

```diff
@@ -23,18 +23,14 @@ const ShapeProps = {
 export const ShapeComponent = (
   props: InferWidgetProps<typeof ShapeProps>
 ): JSX.Element => {
-  const newStyle: any = {
+  const style = {
+    ...useCommonCss(props),
     width: props.shapeWidth ? props.shapeWidth : "100%",
     height: props.shapeHeight ? props.shapeHeight : "100%",
     borderRadius: props.shapeRadius ? props.shapeRadius : "",
-    transform: props.shapeTransform ? props.shapeTransform : "",
-    ...props.border?.css()
+    transform: props.shapeTransform ? props.shapeTransform : ""
```

Comment:
Very minor but I think it's nice to use the [coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) operator, saves having to write props.shapeTransform twice.
`props.shapeTransform ? props.shapeTransform : ""` -> `props.shapeTransform ?? ""`